hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/exceptions/OMException.java

```diff
@@ -259,6 +258,8 @@ public enum ResultCodes {
 
     FEATURE_NOT_ENABLED,
 
-    INVALID_SNAPSHOT_ERROR
+    INVALID_SNAPSHOT_ERROR,
+
+    BUCKET_SNAPSHOT_EXISTS,
```

Comment:
Maybe we can have a more generic name for this? Like `SNAPSHOT_EXISTS`, since we would have snapshot enabled on the volume level as well. Or we can worry about this later.

Reply:
Done.